notebooks/GWAS_1000_genomes_p1/main.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python
+# coding: utf-8
```

Comment:
This is a python2 convention.  All python3 scripts default to `utf-8`, so this isn't necessary.  I was recently doing this too.  >.>

Reply:
I just realized I do this because I had an edge case on a personal project involving CJK characters, like, a year ago... Old habits die hard.

Reply:
Addressed in [this commit](https://github.com/DataBiosphere/bdcat_notebooks/commit/5f1e293fb93dfedd7d6fb8cb14b7b94f57489072) on new branch. Will open a seperate PR to remove from the blood pressure notebooks.